<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogProAdd</class>
 <widget class="QDialog" name="DialogProAdd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>425</width>
    <height>611</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>570</y>
     <width>167</width>
     <height>24</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>52</y>
     <width>381</width>
     <height>501</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="3">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>项目编号：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="CNo"/>
      </item>
     </layout>
    </item>
    <item row="1" column="0" colspan="3">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>项目名称：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="projectName"/>
      </item>
     </layout>
    </item>
    <item row="2" column="0" colspan="3">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>经费（元）：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="fund"/>
      </item>
     </layout>
    </item>
    <item row="3" column="0" colspan="3">
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>参与人数：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="peopleNum"/>
      </item>
     </layout>
    </item>
    <item row="4" column="0" colspan="3">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>立项时间：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="startTime"/>
      </item>
     </layout>
    </item>
    <item row="5" column="0" colspan="3">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>结题时间：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="endTime"/>
      </item>
     </layout>
    </item>
    <item row="6" column="0" colspan="3">
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>完成评价：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="judgement">
        <item>
         <property name="text">
          <string>优</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>良</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>中</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>差</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>未能正常结题</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item row="7" column="0" colspan="3">
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>负责人：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leadMan"/>
      </item>
     </layout>
    </item>
    <item row="8" column="0" colspan="3">
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>负责人电话：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leadPhoneNum"/>
      </item>
     </layout>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>成果形式：</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QComboBox" name="productForm">
      <item>
       <property name="text">
        <string>软件</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>论文</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>作品</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>其他</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="9" column="2">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="10" column="0" colspan="3">
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>成果名称：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="productName"/>
      </item>
     </layout>
    </item>
    <item row="11" column="0" colspan="3">
     <layout class="QHBoxLayout" name="horizontalLayout_11">
      <item>
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>是否支持：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="isSupport">
        <item>
         <property name="text">
          <string>Yes</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>No</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>126</width>
     <height>17</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_12">
    <item>
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>序号：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="number">
      <property name="text">
       <string>待定</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogProAdd</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogProAdd</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
